README.md

```diff
@@ -3,7 +3,61 @@
 It's an initial Java project to connect to Blibli Seller API. 
 Please feel free to relay your comments, suggestions or corrections through pull request.
 
-Installation step:
-1. Download/clone the initial project
-2. Look at 'App.java' as a main class. It show you to use the client's code
-3. Or download the final .jar here https://github.com/bliblidotcom/seller-api-client-java/releases/tag/1.0.0-1
+### Setup
+
+<hr>
+
+##### Maven
+
+```xml
+<!-- Add seller api client dependency -->
+<dependency>
+    <groupId>com.blibli.oss.sellerapi.client</groupId>
+    <artifactId>seller-api-client</artifactId>
+    <version>${seller-api-client.version}</version>
+</dependency>
+
+<!-- Set seller api client version -->
+<properties>
+    <seller-api-client.version>...</seller-api-client.version>
+</properties>
+
+<!-- add blibli bintray repository --> 
+<repositories>
+    <repository>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>bintray-bliblidotcom-maven</id>
+      <name>bintray</name>
+      <url>https://dl.bintray.com/bliblidotcom/maven</url>
+    </repository>
+</repositories>
+```
+
+##### JAR
+
+> [Release note](https://github.com/bliblidotcom/seller-api-client-java/releases)
+
+- v1.2.0-1 : https://github.com/bliblidotcom/seller-api-client-java/releases/tag/1.2.0-1
+- v1.1.0-1 : https://github.com/bliblidotcom/seller-api-client-java/releases/tag/1.1.0-1
+
+### Code Sample
+
+<hr>
+
```

Comment:
Option 3: Copy the codes into your project.
To understand how the code works, you can simply run the App.java or AppBasicAuth.java.
Then you can dive into the codes and copy it as your needs.